PhysicsTools/TensorFlow/src/NTSession.cc

```diff
@@ -12,13 +12,10 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-//NOTE: The memory layout of the Node class changes depending on if NDEBUG was
-// set when tensorflow was compiled. The reason is Node class holds two edgeset
-// class instances and edgeset adds a member data if NDEBUG is set
```

Comment:
the NDEBUG define is not in the upstream.
Should this comment be still kept (perhaps to be moved to the part following "this file is an adaptation"?

Reply:
Yes, since we use it, we should keep the note here.